modules/pv.py

```diff
@@ -102,19 +126,47 @@ def pv_action(self):
             self.module.exit_json(msg="Nothing to do")
         return self.get_volume_command(self.disks)
 
+
     def get_volume_command(self, disk):
+        self.module.exit_json("Warning! Disks is deprecated. Use disk")
```

Comment:
We should print this message only when the user has provided `disks' in this case we are printing all the time.